<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>593</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>593</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>593</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LifeGame</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/48pix/B_stat48_element_share.png</normaloff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/48pix/B_stat48_element_share.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(122, 122, 122);
border-radius:40;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(122, 122, 122);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="play_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(64, 140, 255);
color: rgb(255, 255, 255);
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20;
</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="one_step_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(64, 140, 255);
color: rgb(255, 255, 255);
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20;
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clean_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(64, 140, 255);
color: rgb(255, 255, 255);
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/120pix/W_stat120_navigation_refresh.png</normaloff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/120pix/W_stat120_navigation_refresh.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="random_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(64, 140, 255);
color: rgb(255, 255, 255);
font: 75 24pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/120pix/W_stat120_element_loader_1.png</normaloff>../../Дизигн/icons/Mercury_Icon_Set/Mercury_Static_icons/PNG/120pix/W_stat120_element_loader_1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
